layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_%.bbappend

```diff
@@ -347,6 +358,11 @@ do_deploy_append_photon-nano() {
     cp ${WORKDIR}/tegra210-nano-cti-NGX003.dtb "${DEPLOYDIR}"
 }
 
+do_deploy_append_photon-tx2-nx() {
+    cp ${WORKDIR}/tegra186-tx2-nx-cti-NGX003.dtb "${DEPLOYDIR}"
+    cp ${WORKDIR}/tegra186-tx2-nx-cti-NGX003-IMX219-2CAM.dtb "${DEPLOYDIR}"
+}
```

Comment:
Let's ship these dtbs with the jetson-dtbs package so they are available in the rootfs boot directory: https://github.com/balena-os/balena-jetson/blob/master/layers/meta-balena-jetson/recipes-kernel/linux/jetson-dtbs.bb

Reply:
Ok sounds good. I looked at the photon-nano as example and they aren't included for that device.